### PR TITLE
python3Packages.wsproto: 0.14.1 -> 0.15.0

### PR DESCRIPTION
**Pull request by jonringer**
###### Motivation for this change
closes #67124

keep python2 version building for the ~10 people that still use it :)

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

```
https://github.com/NixOS/nixpkgs/pull/67303
5 package were build:
mitmproxy python37Packages.fastapi python37Packages.starlette python37Packages.uvicorn python37Packages.wsproto
```

### CODE REVIEW

File: pkgs/development/python-modules/wsproto/default.nix [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
+{ lib, buildPythonPackage, fetchPypi, isPy27
+, h11
+, pytest
+}:
 
 buildPythonPackage rec {
   pname = "wsproto";
-  version = "0.14.1";
+  version = "0.15.0";
+  disabled = isPy27;
```
Comment (lsix) [inline pkgs/development/python-modules/wsproto/default.nix]: Apparently, python-3.5 support is removed in release 0.15.0, and we have a `python35Packages` packages-set in `nixpkgs` (see https://github.com/python-hyper/wsproto/blob/master/CHANGELOG.rst#0150-2019-08-10).

I think we should prefer something like `disabled = pythonOlder "3.6"`. And of course, the same goes in `python-packages.nix`.

Comment (jonringer) [reply pkgs/development/python-modules/wsproto/default.nix]: you're correct sir, updating :)

Comment (jonringer) [reply pkgs/development/python-modules/wsproto/default.nix]: updated, 2.7 && 3.5 -> 0.14.1 and 3.6 && 3.7 -> 0.15.0

**Comment by jonringer** [comment]
@GrahamcOfBorg build python27Packages.wsproto python35Packages.wsproto python36Packages.wsproto python37Packages.wsproto

**Comment by lsix** [comment]
:+1:  Thanks for the PR.

**Pull request merged**